HADotNet.Core/Models/AutomationObject.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace HADotNet.Core.Models
+{
+    /// <summary>
+    /// Represents automation object.
+    /// </summary>
+    public class AutomationObject
+    {
+        /// <summary>
+        /// Gets or sets the ID.
+        /// </summary>
+        [JsonProperty("id")]
+        public string Id { get; set; }
+
+        /// <summary>
+        /// Gets or sets the alias.
+        /// </summary>
+        [JsonProperty("alias")]
+        public string Alias { get; set; }
+
+        /// <summary>
+        /// Gets or sets the description.
+        /// </summary>
+        [JsonProperty("description")]
+        public string Description { get; set; }
+
+        /// <summary>
+        /// Gets or sets the actions.
+        /// </summary>
+        [JsonProperty("action")]
+        public List<Dictionary<string, object>> Actions { get; set; }
+
+        /// <summary>
+        /// Gets or sets the conditions.
+        /// </summary>
+        [JsonProperty("condition")]
+        public List<Dictionary<string, object>> Conditions { get; set; }
+
+        /// <summary>
+        /// Gets or sets the triggers.
+        /// </summary>
+        [JsonProperty("trigger")]
+        public List<Dictionary<string, object>> Triggers { get; set; } = new List<Dictionary<string, object>>();
```

Comment:
Just curious why this is initialized as empty while `Actions` and `Conditions` are not? Was there an issue this addressed?

Reply:
It sound like interesting behavior of the API, but you can create an automation by not passing conditions and actions, but triggers should be an empty array, even if you don't have any triggers and want your automation to be run manually... It caused me some confusion to find out this behavior, so I added this piece of code to save somebodies time in the future.